### PR TITLE
local completed

### PR DESCRIPTION
**Pull request by irionura52**
cron job + 기존 BE&FE 연결 확인 후 merge

크게 달라진것 없이 train_model.py에서 환경변수만 수정해서 다시 pr 올립니다.

### CODE REVIEW

File: auto_run/run_training.sh [hunk 1]
```diff
@@ -1,15 +1,37 @@
...
+LOG_DIR=logs
+mkdir -p "${LOG_DIR}"
+
+# 작업 디렉토리(절대경로) 보장
+cd "$(dirname "$0")"/..
+
+# 재학습 시작 (uv run 으로 환경 lock & sync 포함)
+echo "▶ TRAIN START: ${TIMESTAMP}"
+uv run python -u src/train_model.py \
+  --epochs 1 \
```
Comment (ljy2855) [inline auto_run/run_training.sh:15]: 에폭 한번만 돌려도 충분한가요?

File: src/data_loader.py [hunk 2]
```diff
@@ -1,19 +1,118 @@
...
+    mnist_root: str = "data"
+) -> Tuple[DataLoader, DataLoader]:
+    """
+    MNIST 데이터만 사용하는 DataLoader를 반환.
+    """
+    transform = transforms.Compose([
+        transforms.ToTensor(),
+        transforms.Normalize((0.1307,), (0.3081,))
+    ])
+    mnist_train = datasets.MNIST(root=mnist_root, train=True, download=True, transform=transform)
```
Comment (ljy2855) [inline src/data_loader.py:21]: 여기서 폴더에 데이터셋이 없으면 다운로드 받게 할 수 있는걸로 이해했는데,
위에 data/raw 는 gitignore에 추가할 수 있을까요?

File: src/eval.py [hunk 3]
```diff
@@ -1,17 +1,18 @@
 import torch
-import torch.nn as nn
-from torch.utils.data import DataLoader
 
-
-def evaluate(model: nn.Module, dataloader: DataLoader, device: torch.device) -> float:
+def evaluate(model: torch.nn.Module, dataloader, device) -> float:
```
Comment (ljy2855) [inline src/eval.py:3]: 평가 지표를 `accuracy` 와 `f1_score`를 쓴다고 했던 것 같은데, f1_score는 따로 측정하지 않나요?

File: tests/best_model.pth
Comment (ljy2855) [inline tests/best_model.pth:1]: 요것도 gitignore에 추가해주세요~

**Review by ljy2855 — COMMENTED** [review]
수고하셨슴니다~ 내용 한번만 봐주세요~

File: auto_run/run_training.sh (hunk 1 above)
Comment (irionura52) [reply auto_run/run_training.sh:15]: 어짜피 다 f1 0.98~0.99라서 상관 없긴래 1로 테스트했는데
5로 바꿨습니다!

File: src/data_loader.py (hunk 2 above)
Comment (irionura52) [reply src/data_loader.py:21]: 넵! src/data/MNIST/랑 data/도 추가하겠습니다

File: src/eval.py (hunk 3 above)
Comment (irionura52) [reply src/eval.py:3]: train_model.py에서 val_acc(eval.py에서 쓰는) 따로 , f1 score 따로 측정하고 있어요.
model 성능은 f1 score 기준으로 best model 선정하고 있는데 eval.py 기준으로 바꿀까요?
